modules/openlmis-web/src/main/webapp/public/js/distribution/model/adult-coverage.js

```diff
@@ -122,22 +156,31 @@ function AdultCoverageLineItem(lineItem) {
   this.healthCenterTetanus2To5 = this.healthCenterTetanus2To5 || {};
 }
 
-function setNotRecorded(field) {
-  if (field) {
-    delete field.value;
-    field.notRecorded = true;
+function setNotRecorded(field, notRecordedApplied) {
+  if(!notRecordedApplied) {
```

Comment:
Rather than introduce a nested conditional, this may be more readable if field.notRecorded were set to notRecordedApplied or !notRecordedApplied. For example:

if (field) {
    …
    field.notRecorded = notRecordedApplied;
}